docs/usage/setup-azure-devops.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Azure DevOps Setup
+description: How to setup Renovate for Azure DevOps
+---
+
+# Azure DevOps Setup
+
+1. Create a brand new pipeline within Azure DevOps, and select your source.
+
+![Azure DevOps create new pipeline](assets/images/azure-devops-setup-1.png)
+
+1. Select your repository.
+
+1. Within <i>Configure your pipeline</i> select: <b>Starter pipeline file</b>.
+
+![Azure DevOps starter pipeline template](assets/images/azure-devops-setup-2.png)
+
+1. Replace all contents with next template (change all <i>PLACEHOLDER</i> strings with your values):
```

Comment:
Is it worth mentioning that this template is specific to npm/yarn projects?

Reply:
> Is it worth mentioning that this template is specific to npm/yarn projects?